docs/src/misc.rst

```diff
@@ -126,6 +126,10 @@ Data types
                 struct sockaddr_in netmask4;
                 struct sockaddr_in6 netmask6;
             } netmask;
+            union {
+                struct sockaddr_in broadcast4;
+                struct sockaddr_in6 broadcast6;
```

Comment:
Since IPv6 doesn't support broadcast (at least, not at this interface address level), I think it makes more sense not to define this portion of the fields.
```suggestion
```

Reply:
Done